pkg/destroy/aws/aws.go

```diff
@@ -1597,18 +1601,40 @@ func deleteS3(session *session.Session, arn arn.ARN, logger logrus.FieldLogger)
 		Bucket: aws.String(arn.Resource),
 	})
 	err := s3manager.NewBatchDeleteWithClient(client).Delete(aws.BackgroundContext(), iter)
```

Comment:
also looks like the only error you'll receive here is [`s3manager.BatchError`](https://github.com/aws/aws-sdk-go/blob/master/service/s3/s3manager/batch.go#L353) ???

Reply:
that implements awserr.Error

Reply:
+1